help/sources/tutorials/ui/create/consent-and-preferences/onetrust.md

```diff
@@ -6,4 +6,73 @@ topic-legacy: overview
 type: Tutorial
 description: Learn how to create a OneTrust source connection using the Adobe Experience Platform UI.
 ---
-# Create a [!DNL OneTrust] source connection in the UI
\ No newline at end of file
+# Create a *ONETRUST* source connection in the UI
+
+This tutorial provides steps for creating a *ONETRUST* source connector using the Platform user interface.
+
+## Overview
+
+*This describes how to create a live outbound connection to Adobe Experience Platform, to ingest both historic and scheduled consent data into Experience Platform through UI experience.
+To read further about OneTrust, please visit https://www.onetrust.com/company/about-us/*
+
+>[!IMPORTANT]
+>
+>This documentation page was created by the *ONETRUST* team. For any inquiries or update requests, please contact them directly at *https://my.onetrust.com/s/contactsupport?language=en_US*.
+
+## Prerequisites
+
+* *Needing to be aware of API key/client credentials generation through OneTrust website. See the [[!DNL OneTrust] OAuth 2 guide](https://developer.onetrust.com/docs/api-docs-v3/b3A6MjI4OTUyOTc-generate-access-token) for detailed instructions on finding your access token.*
+
+>[!IMPORTANT]
+>
+>The access token does not refresh automatically when expires because system-to-system refresh tokens are not supported by OneTrust.
+Therefore it is necessary to make sure that the access tokens are updated in the connection before it expires. The maximum life of an access token that can be configured is 1 Year.
+
+### Gather required credentials
```

Comment:
Hey @jmaniot thanks for doing this. Only comment is can we update the credentials outlined here to reflect the ones used in the UI, so: host, authorization test URL, and access token instead of API Key and Client Credentials. Thanks!

Reply:
Thanks @carlojavier . Updated with the required information. Added Host and Access Token details which are mandatory attributes for the connection.